charts/enforcer-k8s/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
we should only skip creating the service account if `serviceAccount.create` is false, right?  we'd still want the RBAC in place